components/host-dashboard/reports-section/TotalMoneyManagedSection.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dollar } from '@styled-icons/boxicons-regular/Dollar';
+import { InfoCircle } from '@styled-icons/boxicons-regular/InfoCircle';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+
+import { formatCurrency } from '../../../lib/currency-utils';
+
+import Container from '../../Container';
+import { Flex } from '../../Grid';
+import StyledCard from '../../StyledCard';
+import StyledTooltip from '../../StyledTooltip';
+import { P, Span } from '../../Text';
+
+const FundAmounts = styled.div`
+  width: 100%;
+  height: 48px;
+  border-radius: 10px;
+  background-color: #9a7bf1;
+  border-right: 350px solid #46347f;
+  padding-top: 10px;
+  padding-left: 5px;
+
+  @media (max-width: 1025px) {
+    height: 130px;
+    background-color: #9a7bf1;
+    border-right: 0px;
+    border-bottom: 40px solid #46347f;
+  }
+`;
+
+const TotalFundsLabel = styled(Container)`
+  display: table-cell;
+  padding-left: 10px;
+  height: 26px;
+  border-radius: 5px;
+  background-color: white;
+  opacity: 80%;
+  vertical-align: middle;
+`;
+
+const TotalMoneyManagedSection = ({ currency, hostMetrics }) => {
+  const {
+    totalMoneyManaged,
+    pendingPlatformFees,
+    pendingPlatformTips,
+    pendingHostFeeShare,
+    platformTips,
+    hostFees,
+    platformFees,
+  } = hostMetrics;
+  const currentAmount = totalMoneyManaged.value * 100;
```

Comment:
We have a field on `Amount` to fetch this directly
```suggestion
  const currentAmount = totalMoneyManaged.valueInCents;
```